File: ProjectNewJava/app/src/main/java/com/example/projectnewjava/Opt_Speaker.java
package com.example.projectnewjava;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import androidx.appcompat.app.AppCompatActivity;

public class Opt_Speaker extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_opt_speaker);

        Button bluetooth = (Button) findViewById(R.id.bluetooth_btn_speaker);

        bluetooth.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent myintent = new Intent(view.getContext(), MainActivityBluetooth.class);
                view.getContext().startActivity(myintent);
            }
        });
    }
}

File: project_trial1/app/src/main/java/com/example/project_trial1/Opt_Speaker.kt
package com.example.project_trial1

import android.content.Intent
import android.os.Bundle
import android.widget.ImageButton
import androidx.appcompat.app.AppCompatActivity

class Opt_Speaker : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_opt_speaker)

        var bluetooth: ImageButton

        bluetooth = findViewById(R.id.bluetooth_btn_speaker)

        bluetooth.setOnClickListener{
            val myintent = Intent(this, MainActivityBluetooth::class.java)
            startActivity(myintent)
        }

    }
}